ddtrace/contrib/django/patch.py

```diff
@@ -352,8 +354,27 @@ def wrapped_factory(func, instance, args, kwargs):
     return func(*args, **kwargs)
 
 
-def _block_request_callable(span):
+def _set_block_tags(request, request_headers, span):
+    try:
+        span.set_tag_str(http.STATUS_CODE, "403")
+        span.set_tag_str(http.METHOD, request.method)
+        url = utils.get_request_uri(request)
+        query = request.META.get("QUERY_STRING", "")
+        _set_url_tag(config.django, span, url, query)
+        if query and config.django.trace_query_string:
+            span.set_tag_str(http.QUERY_STRING, query)
+        user_agent = _get_request_header_user_agent(request_headers)
+        if user_agent:
+            span.set_tag_str(http.USER_AGENT, user_agent)
+    except Exception as e:
```

Comment:
There are a lot of different kind of exceptions that can come from this? Wouldn't most be like ValueError or TypeError?